easy-ui-react/src/SearchNav/EmphasizedText.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { ReactNode } from "react";
+import { Text } from "../Text";
+
+export type EmphasizedTextProps = {
+  /**
+   * Emphasized text content to display.
+   */
+  children: ReactNode;
+};
+
+export function EmphasizedText(props: EmphasizedTextProps) {
+  const { children } = props;
+
+  return <Text variant="subtitle1">{children}</Text>;
```

Comment:
**question**: out of curiosity, do you know if it's a design requirement for this text to be uppercase? i'm wondering if we should enforce that in code or keep it as-is to allow that flexibility for the consumer

Reply:
ah good question. confirmed that we basically never want this to be lowercase, added a small wrapper with some css